### PR TITLE
Khanh branch

### PR DESCRIPTION
**Pull request by KHANHHOANG1988**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,19 +1,158 @@
...
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Foundation | Welcome</title>
+    <link rel="stylesheet" href="style.css" />
+    <link rel="stylesheet" href="https://dhbhdrzi4tiry.cloudfront.net/cdn/sites/foundation.min.css">
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+  </head>
+  <body>
+    <script src="logic.js"></script>
```
Comment (ryangautier1) [inline index.html]: Should be at the bottom

File: index.html [hunk 2]
```diff
@@ -1,19 +1,158 @@
...
+    <div class="top-bar-right">
+        <ul class="menu">
+        <li><a href="#">Create an Account</a></li>
+        <li><a class="button">Login</a></li>
+        </ul>
+    </div>
+    </div>
+    <!-- /Navigation -->
 
-    <title>Lunch and Wag</title>
```
Comment (ryangautier1) [inline index.html]: Title should be Lunch and Wag

File: index.html (hunk 1 above)
Comment (KHANHHOANG1988) [reply index.html]: fixed

File: index.html (hunk 2 above)
Comment (KHANHHOANG1988) [reply index.html]: fixed

**Pull request merged**